tests/integration/targets/inventory_vmware_vm_inventory/aliases

```diff
@@ -1,5 +1,4 @@
 shippable/vcenter/group1
 cloud/vcenter
-destructive
 needs/target/prepare_vmware_tests
-zuul/vmware/vcenter_only
+destructive
```

Comment:
We can remove `destructive` since we are not destroying any objects from vCenter environment. I think this was inadvertently added.
Also, keep `zuul/vmware/vcenter_only` since it required for Zuul CI job.